packages/turf-kinks/index.ts

```diff
@@ -42,8 +43,7 @@ export default function kinks<T extends LineString | MultiLineString | Polygon |
         features: [],
     };
 
-    // @ts-ignore
-    if (featureIn.type === "Point" || featureIn.type === 'MultiPoint') {
+    if ((featureIn as GeometryObject | Feature<GeometryObject>).type === "Point" || (featureIn as GeometryObject | Feature<GeometryObject>).type === 'MultiPoint') {
```

Comment:
This still doesn't handle Features correctly, I think you want something like:

```
if (featureIn as GeometryObject).type === "Point" ||
  featureIn as GeometryObject).type === 'MultiPoint' ||
  (featureIn as Feature<GeometryObject>.type === 'Feature' && (
    featureIn.geometry.type === 'Point' ||
    featureIn.geometry.type === 'MultiPoint'))) {
```

Reply:
Think I've sorted this by limiting support to Features rather than Feature | Geometry